getting-started/setup-building.rst

```diff
@@ -483,31 +483,47 @@ Regenerate ``configure``
 
 If a change is made to Python which relies on some POSIX system-specific
 functionality (such as using a new system call), it is necessary to update the
-``configure`` script to test for availability of the functionality.
+:file:`configure` script to test for availability of the functionality.
```

Comment:
```suggestion
:cpy-file:`configure` script to test for availability of the functionality.
```

https://github.com/python/devguide/blob/8ffeda5c27586f0c3cfc7653be6fa41a9cc90de0/_extensions/custom_roles.py#L13-L14

This role would be useful, here and below, to link to the file in the main repo.

Reply:
PTAL